tests/php/Service/RegisteredMethodManagerTest.php

```diff
@@ -66,6 +80,33 @@ public function testRegisterForMemberCreatesNewMethod()
         $this->assertCount(1, $member->RegisteredMFAMethods());
     }
 
+    public function testRegisterForMemberSendsNotification()
+    {
+        /** @var Member&MemberExtension $member */
+        $member = Member::create(['FirstName' => 'Mike', 'Email' => 'test@example.com']);
+        $member->write();
+        $method = new BasicMathMethod();
+
+        $manager = RegisteredMethodManager::singleton();
+        RegisteredMethodManager::singleton()->registerForMember($member, $method, ['foo', 'bar']);
+
+        $this->assertEmailSent($member->Email);
+    }
+
+    public function testRegisterBackupMethodDoesNotSendEmail()
+    {
+        /** @var Member&MemberExtension $member */
+        $member = Member::create(['FirstName' => 'Mike', 'Email' => 'test@example.com']);
+        $member->write();
+        $method = new BackupCodeMethod();
+
+        $manager = RegisteredMethodManager::singleton();
+        RegisteredMethodManager::singleton()->registerForMember($member, $method, ['foo', 'bar']);
+
+        $this->assertNull(static::findEmail($member->Email));
```

Comment:
```suggestion
        $this->assertNull($this->findEmail($member->Email));
```

Reply:
this is a static method

Reply:
Yeah, so are all of PHPUnit's assertions, but that doesn't mean you have to call them statically

Reply:
BTW the reason I mentioned this is because `assertEmailSent()` is static and you're calling it non-statically in the same class

Reply:
Oh 😅

Reply:
I updated all static calls to be statically called.

Reply:
Oh, oh.
Let me change again.